stubs/synapse/synapse_rust/push.pyi

```diff
@@ -54,3 +54,6 @@ class PushRuleEvaluator:
         user_id: Optional[str],
         display_name: Optional[str],
     ) -> Collection[Union[Mapping, str]]: ...
+    def matches(
+        self, condition: JsonDict, user_id: Optional[str], display_name: Optional[str]
+    ) -> bool: ...
```

Comment:
I was attempting to write some tests and was trying to double check the signature, Cmd+clicked in PyCharm, which brought me to the autogenerated Python code. Then I tried to find the stub and couldn't.

I think unignoring mypy from the test files might have picked it up? Not 100% sure on that though since I assume we use this in production code?

I do know adding an incorrect type here *is* used, so the stubs are being used somehow.

Reply:
> I think unignoring mypy from the test files might have picked it up? Not 100% sure on that though since I assume we use this in production code?

This generated the following:

```
tests/push/test_push_rule_evaluator.py:78: error: "PushRuleEvaluator" has no attribute "matches"  [attr-defined]
```